app/components/election-urgent-live-filters.js

```diff
@@ -0,0 +1,52 @@
+'use-strict'
+
+// https://github.com/EnCiv/undebate-ssp/issues/21
+
+import cx from 'classnames'
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+
+import SvgElectionUrgentFilter from '../svgr/election-urgent-filter'
+import SvgElectionLiveFilter from '../svgr/election-live-filter'
+
+function ElectionUrgentLiveFilters(props) {
+    const { onDone, name = 'election-urgent-live-filters', style, className } = props
+    const classes = useStyles()
+
+    const filterType = e => {
+        onDone({ value: e.currentTarget.id, valid: true })
+    }
+
+    return (
+        <div>
+            <div className={classes.circleContainer}>
+                <SvgElectionUrgentFilter id='urgentFilter' className={classes.circle} onClick={filterType} />
```

Comment:
I used an Id because I needed to determine whether the user clicked on the urgent filter (circle) or the live filter (purple). I didn't see anything in the event object that would provide that information so hence the use of the id. Maybe I'm missing something? Is there something in the event object that would tell me which circle the clicked on?

Reply:
Ok...Got it. I changed that.